src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml

```diff
@@ -22,6 +22,28 @@
     <!-- BootStrap Select -->
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/css/bootstrap-select.min.css"/>
     <link id="theme-css" rel="stylesheet" href="~/site.css" asp-append-version="true" />
+    @functions {
+        public string getModelType()
+        {
+            return ViewData.ModelMetadata.ModelType.Name;
+        }
+    }
+    @{
+        var modelType = getModelType();
```

Comment:
I wonder if it is possible to replace this line as follows (and then drop the `getModelType()` function definition above?
```suggestion
        var modelType = ViewData.ModelMetadata.ModelType.Name;
```

Reply:
Tested & committed